docs/2.guide/3.going-further/5.nuxt-app.md

```diff
@@ -21,12 +21,12 @@ function useMyComposable () {
 Plugins also receive `nuxtApp` as the first argument for convenience. [Read more about plugins.](/docs/guide/directory-structure/plugins)
 
 ::alert{icon=👉}
-**`useNuxtApp` (on the server) only works during `setup`, inside Nuxt plugins or `Lifecycle Hooks`**.
+**`useNuxtApp` (on the server) only works during `setup` inside Nuxt plugins or `Lifecycle Hooks`**.
```

Comment:
Removing comma changed the meaning of the sentence :)

Also, worth having a look: https://www.proofreadnow.com/blog/putting-only-in-its-place
```suggestion
**`useNuxtApp` (on the server) works only during component's `setup` function, lifecycle hooks and inside Nuxt plugins.**
```